### PR TITLE
WELD-1896 Improve ConversationLogger.unableToLoadCurrentConversations()

### PR DESCRIPTION
**Pull request by mkouba**

### CODE REVIEW

**Comment by weld-tester** [comment]
Triggering build using a merge of 175442cd4467b0d623427a67c0ab04a35d123a87 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 573 is now running using a merge of 175442cd4467b0d623427a67c0ab04a35d123a87 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/573

**Comment by weld-tester** [comment]
Build 573 outcome was SUCCESS using a merge of 175442cd4467b0d623427a67c0ab04a35d123a87 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/573
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/573

**Comment by weld-tester** [comment]
Build 573 outcome was SUCCESS using a merge of 175442cd4467b0d623427a67c0ab04a35d123a87 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/573
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/573

**Comment by weld-tester** [comment]
Build 573 outcome was SUCCESS using a merge of 175442cd4467b0d623427a67c0ab04a35d123a87 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/573
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/573

**Comment by weld-tester** [comment]
Build 573 outcome was SUCCESS using a merge of 175442cd4467b0d623427a67c0ab04a35d123a87 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/573
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/573

**Comment by weld-tester** [comment]
Build 573 outcome was SUCCESS using a merge of 175442cd4467b0d623427a67c0ab04a35d123a87 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/573
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/573

**Comment by weld-tester** [comment]
Build 573 outcome was SUCCESS using a merge of 175442cd4467b0d623427a67c0ab04a35d123a87 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/573
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/573

File: impl/src/main/java/org/jboss/weld/context/AbstractConversationContext.java [hunk 1]
```diff
@@ -447,20 +447,24 @@ private void checkIsAssociated() {
     private Map<String, ManagedConversation> getConversationMap() {
         checkIsAssociated();
         checkContextInitialized();
-        if (!(getRequestAttribute(getRequest(), CONVERSATIONS_ATTRIBUTE_NAME) instanceof Map<?, ?>)) {
-            throw ConversationLogger.LOG.unableToLoadCurrentConversations();
+        R request = getRequest();
+        Object attribute = getRequestAttribute(getRequest(), CONVERSATIONS_ATTRIBUTE_NAME);
```
Comment (jharting) [inline impl/src/main/java/org/jboss/weld/context/AbstractConversationContext.java]: reduntant `getRequest()` call

Comment (mkouba) [inline impl/src/main/java/org/jboss/weld/context/AbstractConversationContext.java]: Good catch... thanks.

**Comment by weld-tester** [comment]
Triggering build using a merge of ef2ec931e0a7236fce7bfcebef0e6b54d7f1c02e on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 576 outcome was FAILURE using a merge of ef2ec931e0a7236fce7bfcebef0e6b54d7f1c02e on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/576
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/576

**Comment by weld-tester** [comment]
Build 576 outcome was FAILURE using a merge of ef2ec931e0a7236fce7bfcebef0e6b54d7f1c02e on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/576
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/576

**Comment by weld-tester** [comment]
Build 576 outcome was FAILURE using a merge of ef2ec931e0a7236fce7bfcebef0e6b54d7f1c02e on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/576
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/576

**Comment by mkouba** [comment]
retest this please

**Comment by weld-tester** [comment]
Triggering build using a merge of ef2ec931e0a7236fce7bfcebef0e6b54d7f1c02e on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/

**Comment by weld-tester** [comment]
Build 578 is now running using a merge of ef2ec931e0a7236fce7bfcebef0e6b54d7f1c02e on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/578

**Comment by weld-tester** [comment]
Build 578 outcome was SUCCESS using a merge of ef2ec931e0a7236fce7bfcebef0e6b54d7f1c02e on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/Weld-2.x-pull-player-executor-osprey/578
Public: https://hudson.jboss.org/hudson/job/Weld-2.x-pull-player-executor-osprey/578

**Comment by jharting** [comment]
Merged, thanks!

**Pull request closed without merging**